tasks/cephfs/test_volume_client.py

```diff
@@ -14,7 +14,7 @@ class TestVolumeClient(CephFSTestCase):
     # the VolumeClient, one for mounting the created shares
     CLIENTS_REQUIRED = 3
 
-    def _volume_client_python(self, client, script):
+    def _volume_client_python(self, client, prefix, ns_prefix, script):
```

Comment:
Since we don't usually need to override them, it would be useful to leave prefix and ns_prefix as trailing arguments with =None defaults, for those tests that don't want to override them